client_app/src/styles/shelter/ShiftsModal.css

```diff
@@ -0,0 +1,56 @@
+.emailIcon {
+  color: #1f5156;
+}
+
+.emailButton {
+  background-color: transparent;
+}
+
+.shiftsTableHeader {
+  font-weight: bold;
+}
+
+.shiftsModal {
+  max-height: 75%;
+  max-width: 100%;
+  min-width: 80%;
+  min-height: 50%;
+  background-color: lightgray;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding-left: 1%;
+  padding-top: 1%;
+  padding-right: 1%
+}
+
+.close-btn {
+  background-color: transparent;
+}
+
+.closeIcon {
+  color: black;
+  position: absolute;
+  right: 3%;
+  top: 6%;
```

Comment:
After a bit of digging, here's what I know (thanks to claude.ai)

# CSS Units Comparison: Percentages vs Pixels

## Percentages (%)

### Advantages:
1. Responsive: Sizes adjust relative to parent container or viewport.
2. Flexible: Works well for fluid layouts that adapt to different screen sizes.
3. Scalable: Easier to maintain consistent proportions across different device sizes.

### Best for:
- Responsive layouts
- Fluid designs
- Elements that should scale with their container

### Examples:
- Width of columns in a grid system
- Font sizes that should adjust with viewport
- Margins or paddings that should scale

## Pixels (px)

### Advantages:
1. Precise: Offers exact control over sizes.
2. Consistent: Looks the same across different devices (barring high DPI screens).
3. Predictable: Easier to visualize and implement specific designs.

### Best for:
- Fixed-size elements
- Detailed graphics or icons
- When pixel-perfect design is crucial

### Examples:
- Border widths
- Box shadows
- Fixed-size images or logos

## Considerations:

1. Accessibility: Percentages for font sizes allow users to adjust text size in browsers more easily.

2. High DPI Screens: Pixel values might appear differently on high-resolution displays.

3. Maintainability: Percentage-based layouts can be easier to maintain for responsive designs.

4. Precision vs Flexibility: Choose based on whether you need exact control or flexible scaling.

5. Mixing Units: Often, the best approach is to use a combination of units based on each element's needs.

**So the question is: what's most important for this component - precision (exact control) or flexibility (flexible scaling)?**

Reply:
> After a bit of digging, here's what I know (thanks to claude.ai)
> 
> # CSS Units Comparison: Percentages vs Pixels
> ## Percentages (%)
> ### Advantages:
> 1. Responsive: Sizes adjust relative to parent container or viewport.
> 2. Flexible: Works well for fluid layouts that adapt to different screen sizes.
> 3. Scalable: Easier to maintain consistent proportions across different device sizes.
> 
> ### Best for:
> * Responsive layouts
> * Fluid designs
> * Elements that should scale with their container
> 
> ### Examples:
> * Width of columns in a grid system
> * Font sizes that should adjust with viewport
> * Margins or paddings that should scale
> 
> ## Pixels (px)
> ### Advantages:
> 1. Precise: Offers exact control over sizes.
> 2. Consistent: Looks the same across different devices (barring high DPI screens).
> 3. Predictable: Easier to visualize and implement specific designs.
> 
> ### Best for:
> * Fixed-size elements
> * Detailed graphics or icons
> * When pixel-perfect design is crucial
> 
> ### Examples:
> * Border widths
> * Box shadows
> * Fixed-size images or logos
> 
> ## Considerations:
> 1. Accessibility: Percentages for font sizes allow users to adjust text size in browsers more easily.
> 2. High DPI Screens: Pixel values might appear differently on high-resolution displays.
> 3. Maintainability: Percentage-based layouts can be easier to maintain for responsive designs.
> 4. Precision vs Flexibility: Choose based on whether you need exact control or flexible scaling.
> 5. Mixing Units: Often, the best approach is to use a combination of units based on each element's needs.
> 
> **So the question is: what's most important for this component - precision (exact control) or flexibility (flexible scaling)?**

I'd say the scaling since WWT and GetHelp emphasized being responsive to mobile devices on meetings and so using the percentages gives us better flexibility because if I configure px for a big screen it's not going to scale down as well for a phone and I set a max width for the model and imprecise sizing just gives more padding but doesn't change how the data is laid out, and the rest of the site is using percentages.

Reply:
@Brijitha1609 what are your thoughts on this?

Reply:
I understand the point about maintaining responsiveness, especially for mobile devices. Since the goal is to ensure the design scales dynamically, sticking with percentages and rem for element positioning and font sizes makes sense. Thanks for clarifying, and I agree that it’s important to keep the design responsive, especially after the refactor to avoid hardcoded px values.